.github/workflows/integration-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: Integration tests
+on:
+  pull_request:
+  push:
+    branches: [main]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        api-level: [33] # TODO: add more api levels to test
```

Comment:
This targets Android 13 (which corresponds to API level 33).
If we'd like to test more levels in parallel, they can be added to the list here.

Reply:
Perfect. I may add 34 soon too, since Credential Manager acts a bit differently there.